service/collector/tiller_reachable.go

```diff
@@ -19,16 +19,28 @@ var (
 )
 
 func (c *Collector) collectTillerReachable(ctx context.Context, ch chan<- prometheus.Metric) {
-	c.logger.LogCtx(ctx, "level", "debug", "message", "collecting Tiller reachability")
-
-	err := c.helmClient.PingTiller(ctx)
 	var value float64
+
+	c.logger.LogCtx(ctx, "level", "debug", "message", "collecting Tiller reachability")
+	chartConfigs, err := c.getChartConfigs()
 	if err != nil {
-		c.logger.Log("level", "error", "message", "could not ping Tiller", "stack", fmt.Sprintf("%#v", err))
+		c.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("could not get ChartConfigs"), "stack", fmt.Sprintf("%#v", err))
+	}
+	if err == nil && len(chartConfigs) <= 0 {
```

Comment:
Please do equals. Array indices can't be negative.

```suggestion
	if err == nil && len(chartConfigs) == 0 {
```